src/vehicle.cpp

```diff
@@ -4528,6 +4594,12 @@ veh_collision vehicle::part_collision( int part, const tripoint &p,
     }
 
     int dmg_mod = part_info( ret.part ).dmg_mod;
+    add_msg("dmg: %d", dmg_mod);
+    if( chainsaw_on && part_info( ret.part ).has_flag( "CHAINSAW" ) &&
+        parts[part].hp > 0 ) {
+        dmg_mod *= 5.0;
```

Comment:
Would be better if that added a secondary hit independent of vehicle speed and mass.
Vehicle collisions are already really bad at respecting armors, having `dmg_mod` reach arbitrarily huge numbers will result in giant critters getting obliterated.